### DISCUSSION TITLE
[Feature proposal] : Avatar generator form

### DISCUSSION THREAD

**Discussion by nohehf** [dicebear/dicebear#193] [Ideas]
Hey, not an issue, but I think it would be nice to have a "form" section on the website, so that users can custom create their own avatar and then just copy the api url.
Something like: [get avataaars form](https://getavataaars.com/)
I can maybe work on it (but not immediately, and I don't know svelte).
But maybe we can first discuss how it should be implemented and the features.
Hope this was useful :)

**Comment by FlorianKoerner** [comment]
Hi @nohehf 

We are currently working on an editor. You can find the alpha version here: [editor.dicebear.com](https://editor.dicebear.com/)

**Reply by nohehf** [reply]
Oh great ! Is there any way of getting involved ?

**Comment by FlorianKoerner** [comment]
https://github.com/orgs/dicebear/discussions/286

**Reply by nohehf** [reply]
Nice thank you!
